networkx/algorithms/assortativity/pairs.py

```diff
@@ -7,7 +7,7 @@
 @nx._dispatchable(node_attrs="attribute")
 def node_attribute_xy(G, attribute, nodes=None):
     """Yields 2-tuples of node attribute values for all edges in `G`.
-    
+
     This generator yields, for each edge in `G` incident to a node in `nodes`,
     a 2-tuple of form ``(attribute value,  attribute value)`` for the parameter
     specified node-attribute.  for all edges in `G`.
```

Comment:
my mistake - I left a half sentence "for all edges in `G`." at the end of line 13.